app/scripts/views/index.js

```diff
@@ -45,6 +46,13 @@ define(function (require, exports, module) {
       } else if (this.isInEmailFirstExperimentGroup('treatment') || action === 'email') {
         // let's the router know to use the email-first signin/signup page
         this.notifier.trigger('email-first-flow');
+        const email = this.relier.get('email');
+        if (email) {
+          const TEST_REQUEST_DELAY_MS = this.broker.isAutomatedBrowser() ? 500 : 0;
```

Comment:
Tangential, but seeing this `isAutomatedBrowser` method makes me think we should have some test coverage that asserts `isAutomatedBrowser` returns `false` in normal running. Do we have anything like that?